hydra-node/src/Hydra/HeadLogic.hs

```diff
@@ -1,6 +1,7 @@
 {-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-partial-fields #-}
```

Comment:
Could: In light of new soon to be proposed hydra coding standard you could remove `-Wpartial-fields` from the `.cabal` file instead of disabling it here again.